sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraCommandPostBody.cs

```diff
@@ -43,6 +43,6 @@ public CassandraCommandPostBody(string command, string host)
         /// <summary> If true, stops cassandra before executing the command and then start it again. </summary>
         public bool? CassandraStopStart { get; set; }
         /// <summary> If true, allows the command to *write* to the cassandra directory, otherwise read-only. </summary>
-        public bool? Readwrite { get; set; }
+        public bool? IsReadwrite { get; set; }
```

Comment:
Based on the description, `AllowReadWrite` may be a better name?

Reply:
Good suggestion, renamed to `AllowWrite`